--- src/dictmanagedlg.cpp.str	2009-04-11 16:17:03.000000000 +0200
+++ src/dictmanagedlg.cpp	2009-04-11 16:17:20.000000000 +0200
@@ -78,7 +78,7 @@
                         gtk_message_dialog_new(
                                 GTK_WINDOW(oNetworkAddDlg->window),
                                 (GtkDialogFlags) (GTK_DIALOG_MODAL | GTK_DIALOG_DESTROY_WITH_PARENT),
-                                GTK_MESSAGE_INFO,  GTK_BUTTONS_OK,
+                                GTK_MESSAGE_INFO,  GTK_BUTTONS_OK, "%s",
                                 msg);
                     g_free(msg);
                     gtk_dialog_set_default_response(GTK_DIALOG(message_dlg), GTK_RESPONSE_OK);
--- src/prefsdlg.cpp.str	2009-04-11 16:15:36.000000000 +0200
+++ src/prefsdlg.cpp	2009-04-11 16:16:47.000000000 +0200
@@ -850,7 +850,7 @@
                 gtk_message_dialog_new(
                         GTK_WINDOW(account_dialog),
                         (GtkDialogFlags) (GTK_DIALOG_MODAL | GTK_DIALOG_DESTROY_WITH_PARENT),
-                        GTK_MESSAGE_INFO,  GTK_BUTTONS_OK,
+                        GTK_MESSAGE_INFO,  GTK_BUTTONS_OK, "%s",
                         error_msg);
             gtk_dialog_set_default_response(GTK_DIALOG(message_dlg), GTK_RESPONSE_OK);
             gtk_window_set_resizable(GTK_WINDOW(message_dlg), FALSE);
@@ -885,7 +885,7 @@
 
 	GtkWidget *message_dlg = gtk_message_dialog_new(GTK_WINDOW(window),
 			(GtkDialogFlags) (GTK_DIALOG_MODAL | GTK_DIALOG_DESTROY_WITH_PARENT),
-			GTK_MESSAGE_INFO, GTK_BUTTONS_OK, msg);
+			GTK_MESSAGE_INFO, GTK_BUTTONS_OK, "%s", msg);
 	gtk_dialog_set_default_response(GTK_DIALOG(message_dlg), GTK_RESPONSE_OK);
 	gtk_window_set_resizable(GTK_WINDOW(message_dlg), FALSE);
 	g_signal_connect_swapped (message_dlg, "response", G_CALLBACK (gtk_widget_destroy), message_dlg);
@@ -953,7 +953,7 @@
                 gtk_message_dialog_new(
                         GTK_WINDOW(register_dialog),
                         (GtkDialogFlags) (GTK_DIALOG_MODAL | GTK_DIALOG_DESTROY_WITH_PARENT),
-                        GTK_MESSAGE_INFO,  GTK_BUTTONS_OK,
+                        GTK_MESSAGE_INFO,  GTK_BUTTONS_OK, "%s",
                         error_msg);
             gtk_dialog_set_default_response(GTK_DIALOG(message_dlg), GTK_RESPONSE_OK);
             gtk_window_set_resizable(GTK_WINDOW(message_dlg), FALSE);
@@ -1349,7 +1349,7 @@
 				gtk_message_dialog_new(
 															 GTK_WINDOW(searchwebsite_add_dialog),
 																 (GtkDialogFlags) (GTK_DIALOG_MODAL | GTK_DIALOG_DESTROY_WITH_PARENT),
-															 GTK_MESSAGE_INFO,	GTK_BUTTONS_OK,
+															 GTK_MESSAGE_INFO,	GTK_BUTTONS_OK, "%s",
 															 error_msg);
 			
 			gtk_dialog_set_default_response(GTK_DIALOG(message_dlg), GTK_RESPONSE_OK);
--- src/stardict.cpp.str	2009-04-11 16:14:37.000000000 +0200
+++ src/stardict.cpp	2009-04-11 16:15:13.000000000 +0200
@@ -1763,7 +1763,7 @@
         gtk_message_dialog_new(
                 parent,
                 (GtkDialogFlags) (GTK_DIALOG_MODAL | GTK_DIALOG_DESTROY_WITH_PARENT),
-                GTK_MESSAGE_INFO,  GTK_BUTTONS_OK,
+                GTK_MESSAGE_INFO,  GTK_BUTTONS_OK, "%s",
                 error_msg);
     gtk_dialog_set_default_response(GTK_DIALOG(message_dlg), GTK_RESPONSE_OK);
     gtk_window_set_resizable(GTK_WINDOW(message_dlg), FALSE);
@@ -1865,7 +1865,7 @@
 			gtk_message_dialog_new(
 				parent,
 				(GtkDialogFlags) (GTK_DIALOG_MODAL | GTK_DIALOG_DESTROY_WITH_PARENT),
-				GTK_MESSAGE_INFO,  GTK_BUTTONS_OK,
+				GTK_MESSAGE_INFO,  GTK_BUTTONS_OK, "%s",
 				error_msg);
 		gtk_dialog_set_default_response(GTK_DIALOG(message_dlg), GTK_RESPONSE_OK);
 		gtk_window_set_resizable(GTK_WINDOW(message_dlg), FALSE);
--- stardict-plugins/stardict-update-info-plugin/stardict_update_info.cpp.str	2009-04-11 16:30:32.000000000 +0200
+++ stardict-plugins/stardict-update-info-plugin/stardict_update_info.cpp	2009-04-11 16:31:00.000000000 +0200
@@ -51,7 +51,7 @@
 	content += "\n";
 	content += latest_news;
 	
-	GtkWidget *dialog = gtk_message_dialog_new(GTK_WINDOW(plugin_info->pluginwin), GTK_DIALOG_MODAL, GTK_MESSAGE_INFO, GTK_BUTTONS_CLOSE, content.c_str());
+	GtkWidget *dialog = gtk_message_dialog_new(GTK_WINDOW(plugin_info->pluginwin), GTK_DIALOG_MODAL, GTK_MESSAGE_INFO, GTK_BUTTONS_CLOSE, "%s", content.c_str());
 	gtk_window_set_title (GTK_WINDOW (dialog), _("Update information"));
 	gtk_dialog_run(GTK_DIALOG(dialog));
 	gtk_widget_destroy (dialog);
@@ -175,7 +175,7 @@
 		std::string content = version_msg_content;
 		content += "\n\n";
 		content += _("Visit StarDict website now?");
-		GtkWidget *dialog = gtk_message_dialog_new(GTK_WINDOW(plugin_info->mainwin), GTK_DIALOG_MODAL, GTK_MESSAGE_INFO, GTK_BUTTONS_YES_NO, content.c_str());
+		GtkWidget *dialog = gtk_message_dialog_new(GTK_WINDOW(plugin_info->mainwin), GTK_DIALOG_MODAL, GTK_MESSAGE_INFO, GTK_BUTTONS_YES_NO, "%s", content.c_str());
 		GtkWidget *prompt = gtk_check_button_new_with_mnemonic(_("_Don't show this until the next update."));
 		gtk_widget_show(prompt);
 		gtk_container_add (GTK_CONTAINER (GTK_DIALOG(dialog)->vbox), prompt);
